assets/lang/sklmessages_tr_TR.properties

```diff
@@ -0,0 +1,69 @@
+lang.name=Türkçe
```

Comment:
Add `Contributors history` like in: https://github.com/skmedix/SKlauncher/blob/master/assets/lang/sklmessages_pl_PL.properties

Reply:
Also please use https://native2ascii.net/ for the whole file